src/domain/relay/contracts/__tests__/proxy-factory-decoder.helper.spec.ts

```diff
@@ -0,0 +1,30 @@
+import { Hex } from 'viem';
+import { faker } from '@faker-js/faker';
+import { ProxyFactoryDecoder } from '@/domain/relay/contracts/proxy-factory-decoder.helper';
+import { createProxyWithNonceEncoder } from '@/domain/relay/contracts/__tests__/proxy-factory-encoder.builder';
+
+describe('ProxyFactoryDecoder', () => {
+  let target: ProxyFactoryDecoder;
+
+  beforeEach(() => {
+    jest.clearAllMocks();
```

Comment:
To be consistent with the recent change [5b0e668](https://github.com/safe-global/safe-client-gateway/commit/5b0e668196b433b1fae7297180ddd52807f821c8)
```suggestion
    jest.resetAllMocks();
```

Reply:
Changed in 192ff518bad0aff21b70ad12ca180db3b701020b.